requirements-wheels.txt

```diff
@@ -2,6 +2,7 @@ click == 8.1.*
 Flask == 2.3.*
 faster-fifo == 1.4.*
 imutils == 0.5.*
+httpx == 0.24.*
```

Comment:
Why introduce a new dependency here? Seems like requests is fine for this use case.

Reply:
python `requests` isn't really thread-safe :(

Reply:
Where are you seeing that? Thread-safe is specifically mentioned in the docs: https://docs.python-requests.org/en/v2.9.1/

What errors are you/others seeing that necessitate this change?